esm-samples/jsapi-custom-widget/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "module": "ESNext",
+    "lib": ["ESNext", "DOM"],
```

Comment:
Can these both use ES2020 instead of ESNext for consistency?

Reply:
Do mean `module` and `lib`?

Reply:
Yes